src/plugins/intel_cpu/src/nodes/memory.cpp

```diff
@@ -159,7 +159,7 @@ MemoryOutputBase::~MemoryOutputBase() {
 }
 
 MemoryInputBase& MemoryOutputBase::getInputNode() {
-    OPENVINO_ASSERT(inputNode, "MemoryOutput ", getName(), " doesn't have sibling input");
+    CPU_NODE_ASSERT(inputNode, "MemoryOutput ", getName(), " doesn't have sibling input");
```

Comment:
Pleaser remove the node type and name from the message.

Reply:
Done

Reply:
Done